hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -20,9 +35,14 @@ const Teams = () => {
             key={index}
             className={styles.teamsListGridItem}
             grid-template-column="true"
-            onClick={() => alert(`Goes to team-detail for team ${team.TeamName}`)}
+            onClick={() => alert(`Goes to team-detail for team ${team.team_code}`)}
```

Comment:
This is a good time to link the teams list page with the team detail page :))))
could you change this onClick to direct the user to /teams/<team code>

Reply:
```
// near the top of the component use the history hook from react router
const history = useHistory()

// in your grid component
onClick={() => history.push(/team/<team code>)}
```

Reply:
It is error 404 rn but that should be okay right?